assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_attribution_drift.py

```diff
@@ -99,12 +105,16 @@ def compute_ndcg_and_write_to_mltable(baseline_explanations, production_explanat
             constants.METRIC_VALUE_COLUMN: baseline_feature[constants.METRIC_VALUE_COLUMN],
             constants.FEATURE_CATEGORY_COLUMN: baseline_feature[constants.FEATURE_CATEGORY_COLUMN],
             constants.METRIC_NAME_COLUMN: "BaselineFeatureImportance",
+            constants.GROUP_COLUMN: production_feature[constants.FEATURE_CATEGORY_COLUMN],
+            constants.GROUP_PIVOT_COLUMN: "",
             constants.THRESHOLD_VALUE: float("nan")}
         production_feature_importance_data = {
             constants.FEATURE_NAME_COLUMN: production_feature[constants.FEATURE_COLUMN],
             constants.METRIC_VALUE_COLUMN: production_feature[constants.METRIC_VALUE_COLUMN],
             constants.FEATURE_CATEGORY_COLUMN: production_feature[constants.FEATURE_CATEGORY_COLUMN],
             constants.METRIC_NAME_COLUMN: "ProductionFeatureImportance",
+            constants.GROUP_COLUMN: production_feature[constants.FEATURE_CATEGORY_COLUMN],
```

Comment:
should this be constant.GROUP_COLUMN: production_feature[constants.GROUP_COLUMN] ?

Reply:
the group for the feature importance is the feature! It's feature importance GROUP BY feature!